Source/Core/Core/HW/AudioInterface.cpp

```diff
@@ -127,6 +127,8 @@ void DoState(PointerWrap& p)
   p.Do(g_AISSampleRate);
   p.Do(g_AIDSampleRate);
   p.Do(g_CPUCyclesPerSample);
+
+  g_sound_stream->GetMixer()->DoState(p);
```

Comment:
May anything here return nullptr?

Reply:
`g_sound_stream` is only nullptr if the boot code hasn't initialized the hardware yet, and `GetMixer()` never returns nullptr as far as I know. So it should be fine.